mrp_bom_hierarchy/README.rst

```diff
@@ -62,6 +62,7 @@ Authors
 ~~~~~~~
 
 * ForgeFlow
+* Grap
 
 Contributors
```

Comment:
I think there is a difference between author and contributor, and I think what you did is a contribution. What do you think?

Reply:
yes totally agree with you ! i change it  !